ut/s3_put_object_action_test.cc

```diff
@@ -63,6 +63,37 @@ using ::testing::DefaultValue;
     action_under_test->fetch_object_info();                               \
   } while (0)
 
+#define CREATE_BUCKET_METADATA_VER                            \
```

Comment:
Can we rename this macro to something e.g. SET_BUCKET_VERSIONING_STATUS_EXPECTATIONS

Reply:
sure

Reply:
done